test/parallel/test-cluster-inspector-debug-brk.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+// Flags: --inspect={PORT}
+// This test ensures that flag --inspect-brk and --debug-brk works properly
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const net = require('net');
+
+const debuggerPort = common.PORT;
+const script = common.fixturesDir + '/empty.js';
+
+function fail() {
+  assert(0); // `node --debug-brk script.js` should not quit
+}
+
+if (cluster.isMaster) {
+
+  function fork(offset, execArgv) {
+    cluster.setupMaster(execArgv);
+    let worker = cluster.fork();
+
+    worker.on('exit', fail);
+
+    let socket = net.connect(debuggerPort + offset, () => {
+      socket.end();
+      worker.removeListener('exit', fail);
```

Comment:
Instead of having this `fail()` function and the `process.on('exit', ...)` block, it might be better to use the cluster worker `exit` event and check the exit code and signal to make sure it was killed by signal (there is an example in the docs).